assets/celebration.svg

```diff
@@ -181,511 +150,228 @@
-        fill="#34A853" />
-    <mask id="mask0" mask-type="alpha" maskUnits="userSpaceOnUse" x="321" y="7" width="26" height="74">
-        <path
-            d="M333.47 78.3367C333.47 78.3367 347.47 58.2503 346.983 12.7213C346.983 12.7213 346.131 8.21713 342.175 7.97366C338.218 7.73019 337.244 12.0518 337.244 12.0518C337.244 12.0518 335.723 44.4942 325.436 62.5111C315.21 80.5279 330.001 83.1452 333.47 78.3367Z"
-            fill="white" />
-    </mask>
-    <g mask="url(#mask0)">
-        <path
-            d="M333.775 29.8861C334.505 27.0253 335.053 24.2863 335.418 21.3646C336.27 15.2779 336.94 9.19109 337.062 3.04346C340.348 3.65213 345.461 4.80862 348.809 5.5999C349.783 14.6692 349.174 23.7993 349.235 32.9295L349.174 33.7207L333.775 30.3122C333.775 30.3122 333.775 30.2513 333.714 30.1296C333.775 30.0078 333.775 29.947 333.775 29.8861Z"
-            fill="#8D6E63" />
-        <path
-            d="M334.627 26.2949C337.853 26.721 341.018 27.0862 344.244 27.4514C345.948 27.634 347.592 27.8775 349.296 28.0601C349.296 29.7035 349.235 31.2861 349.235 32.9295L349.174 33.7208L333.775 30.3122C333.775 30.3122 333.775 30.2513 333.714 30.1296C333.714 30.0687 333.775 30.0078 333.775 29.947C334.079 28.6688 334.383 27.5123 334.627 26.2949Z"
-            fill="#126B26" />
-    </g>
-    <path
-        d="M302.245 80.2845C302.245 80.2845 284.654 63.2415 276.559 18.4429C276.559 18.4429 276.559 13.8779 280.394 12.8431C284.228 11.8084 285.994 15.8865 285.994 15.8865C285.994 15.8865 293.602 47.4159 307.115 63.2415C320.627 79.0671 306.567 84.3017 302.245 80.2845Z"
-        fill="#34A853" />
-    <mask id="mask1" mask-type="alpha" maskUnits="userSpaceOnUse" x="276" y="12" width="37" height="70">
-        <path
-            d="M302.245 80.2845C302.245 80.2845 284.654 63.2415 276.559 18.4429C276.559 18.4429 276.559 13.8779 280.394 12.8431C284.228 11.8084 285.994 15.8865 285.994 15.8865C285.994 15.8865 293.602 47.4159 307.115 63.2415C320.627 79.0671 306.567 84.3017 302.245 80.2845Z"
-            fill="white" />
-    </mask>
-    <g mask="url(#mask1)">
-        <path
-            d="M292.75 32.7469C291.472 30.1296 290.437 27.4515 289.524 24.7124C287.576 18.93 285.75 13.0258 284.411 7.06079C281.307 8.27814 276.498 10.3476 273.394 11.8085C274.185 20.8777 276.437 29.7644 278.142 38.712L278.385 39.5032L292.811 33.2339C292.811 33.2339 292.811 33.173 292.872 32.9904C292.811 32.8687 292.811 32.8078 292.75 32.7469Z"
-            fill="#8D6E63" />
-        <path
-            d="M291.289 29.3992C288.246 30.4339 285.141 31.4078 282.037 32.3817C280.455 32.8686 278.811 33.4164 277.168 33.9034C277.472 35.4859 277.837 37.0685 278.142 38.6511L278.385 39.4423L292.811 33.173C292.811 33.173 292.811 33.1121 292.872 32.9295C292.872 32.8686 292.811 32.8078 292.811 32.7469C292.202 31.6513 291.715 30.5557 291.289 29.3992Z"
-            fill="#126B26" />
+
+    <svg width="374" height="230" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
You are including a <svg> tag inside a <svg> tag (see first lien of the file). You should not do that.=, make sure the SVG markup is valid.